README.md

```diff
@@ -27,7 +27,7 @@ the tasks will be executed the same way years from now regardless what versions
 Cirrus CLI reuses the [same YAML configuration format as Cirrus CI](https://cirrus-ci.org/guide/writing-tasks/) which allows to
 reuse a large [list of examples](https://cirrus-ci.org/examples/) created by Cirrus CI community.
 
-**Note:** Cirrus CLI can be used in any environment that has Docker installed. It can be your laptop or any CI system you already have
+**Note:** Cirrus CLI can be used in any environment that has Docker or Podman installed. It can be your laptop or any CI system you already have
```

Comment:
Links?

```suggestion
**Note:** Cirrus CLI can be used in any environment that has [Docker](https://docker.com/) or [Podman](https://podman.io/) installed. It can be your laptop or any CI system you already have
```